docs/send-data/sumo-logic-distribution-for-opentelemetry-collector/comparison.md

```diff
@@ -0,0 +1,264 @@
+---
+id: comparison
+title: Comparing the Installed Collector and OpenTelemetry Collector
+sidebar_label: OTEL Collector vs. Installed Collector
+---
+
+
+## When to use OpenTelemetry
+
+**Ideal use cases** include when you:
+
+- leverage the **Supported Sources** and **Supported Platforms** listed below
+- are looking for a single agent as opposed to managing multiple agents
+- are having scale issues with FluentD on Kubernetes Collection
+- are looking for ARM support
+
+**Avoid use cases** that:
+
+- are using an **Unsupported Source** or **Unsupported Platform**
+- use CSE due to lack of Windows support
+- require remote management and remote configuration
+- require FIPS support
+- require Ingest Budgets
+- use the Collector Management API (e.g. for Health Events or CRUD operations)
+- require CPU target
+
+## Support Matrix
+
+<table>
+  <tr>
+   <td><strong>Supported Platforms</strong>
+   </td>
+   <td><strong>Unsupported Platforms</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>
+<ul>
+  <li>Linux</li>
+  <li>MacOS</li>
+  <li>Kubernetes</li>
+</ul>
+   </td>
+   <td>
+<ul>
+  <li>Windows OS</li>
```

Comment:
I would move Windows to the supported list

Reply:
That would leave the unsupported column blank - is that correct? @rnishtala-sumo @astencel-sumo 
<img width="404" alt="Screenshot 2023-02-28 at 5 23 55 PM" src="https://user-images.githubusercontent.com/56411016/222020935-7b3eeb3e-fbe6-41cd-a8ff-ffd3c86da30f.png">

Reply:
We can remove `Unsupported Platforms`. We wanted to `emphasize` than Windows is not supported.

Regarding `Windows Collection` we need to test it and prepare migration from installed collector first